speculos/main.py

```diff
@@ -326,6 +346,12 @@ def main(prog=None) -> int:
             args.model = metadata["target"]
             logger.warn(f"Device model detected from metadata: {args.model}")
 
+    # For Nano S and Blue, it can only be BAGL
+    if args.model == "nanos" or args.model == "blue":
+        use_bagl = True
+    else:
+        use_bagl = get_elf_fonts_size(app_path) != 0
```

Comment:
```suggestion
    elif args.model == "stax" or args.model == "europa"
        use_bagl = False
    else:
        use_bagl = check_if_use_bagl(app_path) != 0
```

Reply:
Not necessary, and I prefer to avoid as much as possible checking on product name

Reply:
it makes things faster, but that's maybe not so slow to perform?
and you could do it the other way requesting the check only for NanoX and NanoS+? which can be more future proof?

Reply:
It takes probably less than 1ms and is only performed once before app launch... so not a big deal at all

Reply:
28ms on my computer (fast compared to the CI), but that's indeed not that much, even considering app tests that starts a Speculos instance 100 times.